railgun/pkg/config/config.go

```diff
@@ -117,6 +118,11 @@ func (c *Config) FlagSet() *pflag.FlagSet {
 			"Define the rate (in seconds) in which configuration updates will be applied to the Kong Admin API. (default: %g seconds)",
 			proxy.DefaultSyncSeconds,
 		))
+	flagSet.Float32Var(&c.ProxyTimeoutSeconds, "timeout-seconds", proxy.DefaultSyncSeconds,
```

Comment:
My initial thoughts here is that the flag `--timeout-seconds` is too broad of a flag, as the controller is reasonably complex and one could fathom this "timeout" being associated when one of several bits of functionality?

We already have `--proxy-sync-seconds` which corresponds directly to the Kong proxy by nomenclature, perhaps we should do that here as well:

```suggestion
	flagSet.Float32Var(&c.ProxyTimeoutSeconds, "proxy-timeout-seconds", proxy.DefaultSyncSeconds,
```

Let me know your thoughts? :thinking:

Reply:
sure. let's add this